app/code/Magento/Sales/Block/Order/Email/Items/Order/DefaultOrder.php

```diff
@@ -93,14 +93,17 @@ public function getProductAdditionalInformationBlock()
     }
 
     /**
-     * Get the html for item price
+     * Return item unit price html
      *
      * @param OrderItem $item
      * @return string
      */
-    public function getItemPrice(OrderItem $item)
+    public function getItemPriceHtml($item = null)
```

Comment:
Could we introduce a new method instead of renaming the existing one? Renaming a public method in a class marked as `@api` considered backward-incompatible change.